lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -445,7 +432,7 @@ IRBuilderAsmJs::AddStatementBoundary(uint statementIndex, uint offset)
     return m_statementReader.MoveNextStatementBoundary();
 }
 
-Js::RegSlot IRBuilderAsmJs::GetTypedRegFromRegSlot(Js::RegSlot reg, WAsmJs::Types type)
+uint32 IRBuilderAsmJs::GetTypedRegFromRegSlot(Js::RegSlot reg, WAsmJs::Types type)
```

Comment:
nit: why this change?

Reply:
I wanted to differentiate an IR Reg from a ByteCode typed Reg. 
I thought it didn't really make sense to treat those as RegSlot